docs/articles/authn-authz.md

```diff
@@ -206,44 +206,46 @@ Use the `accessControl` attribute in the configuration file to define a set of i
 "http": {
 ...
   "accessControl": {
-    "**": {
```

Comment:
This doesn't look right.
In v1.4.3, the policy looks like this.
https://github.com/project-zot/zot/blob/v1.4.3/examples/config-policy.json#L16

Reply:
In your latest "What's new" document, we should also call out that the policy configuration has changed as your are pointing out here.

Reply:
This bug issue (https://github.com/project-zot/zot/issues/1673) says that the "repositories" attribute must be under the "accessControl" attribute.  Is this only for 2.0 and not for 1.4.3?

Reply:
Correct, always best to look under examples/ for **that** version

Reply:
We should actually call this out in a "NOTE" topic under what's new

Reply:
Removed 'repositories' under 'accessControl' -- not needed for v1.4.
Will add note about this change in What's New for v2.0.